other/concat-exports.js

```diff
@@ -0,0 +1,88 @@
+const appendToFile = require('fs').appendFile
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+const jsBuiltInObjectNames = [
```

Comment:
I wonder whether there's an npm package (probably by Sindre Sorhus) that has these so we don't have to hard-code them 🤔